build.gradle

```diff
@@ -16,6 +16,8 @@
 
 import com.github.benmanes.gradle.versions.reporter.PlainTextReporter
 
+apply plugin: 'java'
```

Comment:
I think this shouldn't be needed

Reply:
When removed gradle build errors with `Could not find method signature()`. According to documentation `plugin must be declared after java or groovy plugin, otherwise nothing will be registered.`, so I have decided to leave it in.